src/librustc_borrowck/borrowck/mod.rs

```diff
@@ -1326,12 +1330,14 @@ impl<'a, 'tcx> BorrowckCtxt<'a, 'tcx> {
                                                   &cmt_path_or_string,
                                                   capture_span,
                                                   Origin::Ast)
-            .span_suggestion(err.span,
+            .span_suggestion_with_applicability(err.span,
                              &format!("to force the closure to take ownership of {} \
                                        (and any other referenced variables), \
                                        use the `move` keyword",
                                        cmt_path_or_string),
-                             suggestion)
+                             suggestion,
+                             Applicability::Unspecified,
+            )
```

Comment:
Nitpick: Change the indentation so that each argument is on their own line four spaces removed and no more from the function call, and `err.span` on its own line as well.